projects/widgets/docs/linken-van-artikels-aan-events-via-widgets.md

```diff
@@ -0,0 +1,123 @@
+# Linken van artikels aan events via redactionele tools
+
+> Onderstaande functionaliteiten kan je enkel gebruiken met toestemming van publiq. Wil je hiermee aan de slag? Contacteer ons dan op <partnerships@publiq.be>.
+
+## Wat zijn redactionele tools?
+
+Redactionele tools zijn een uitbreiding bovenop de widget modules. Ze zijn gericht naar partners die op hun eigen website artikels publiceren over evenementen.
+
+Met behulp van deze redactionele tools kunnen partners hun artikels eenvoudig automatisch laten linken aan de overeenkomstige evenementen in de UiTdatabank. Eens gelinkt wordt een artikel op verschillende kanalen en websites van UiT gepubliceerd (zoals UiTinVlaanderen).
+
+Door redactionele content te linken aan evenementen zorgen we op verschillende vlakken voor meerwaarde:
+
+* **Redacteurs** bereiken een **groter publiek** voor hun **artikels**, omdat het artikel op tal van websites gepubliceerd wordt.
+* Daarnaast **besparen redacteurs tijd uit**, want aan de hand van de widgets kunnen ze de praktische informatie van een evenement in enkele klikken integreren in artikels, en de informatie is steeds up to date.
+* Dankzij de link tussen artikel en evenement krijgen **activiteiten van organisatoren extra publiciteit**
+* **Geïnteresseerde bezoekers** van activiteiten worden **beter en vollediger geïnformeerd** over een evenement
+
+#### Een voorbeeld
+
+BRUZZ schreef een artikel over de miljoenste bezoeker in Trainworld en de nieuwe tentoonstelling "Horta & Van de Velde - De trein der ontwerpers". De redacteur van dat artikel integreerde een widget om de praktische informatie over dat evenement te tonen: <https://www.bruzz.be/samenleving/train-world-schaarbeek-verwelkomt-miljoenste-bezoeker-2023-01-03>
```

Comment:
```suggestion
BRUZZ schreef een artikel over [de miljoenste bezoeker in Trainworld en de nieuwe tentoonstelling "Horta & Van de Velde - De trein der ontwerpers"](https://www.bruzz.be/samenleving/train-world-schaarbeek-verwelkomt-miljoenste-bezoeker-2023-01-03). De redacteur van dat artikel integreerde een widget om de praktische informatie over dat evenement te tonen:
```

Nit: I can't immediately find a good source to back it up, but I think that it's recommended to integrate links into the text instead of including the complete URL in/after the text.